students/swietczak_monika/lesson_08_regular_expressions/regex_version_of_strip.py

```diff
@@ -0,0 +1,20 @@
+import re
+
+
+def regex_strip(some_string, chars_to_strip):
+    if re.search(r"[\.\*\\]", chars_to_strip) is not None:
+        result = re.sub("^" + re.escape(chars_to_strip) + "*", '', some_string)
+        result = re.sub(re.escape(chars_to_strip) + "*$", '', result)
+    else:
+        result = re.sub(r'^' + chars_to_strip + '*', '', some_string)
+        result = re.sub(chars_to_strip + '*$', '', result)
```

Comment:
You might use OR statement for above two cases:

regex = re.compile(r'^[' + chars_to_strip + ']*|[' + chars_to_strip + ']*$')
re.sub(regex, '', string)